.eslintrc.cjs

```diff
@@ -12,6 +12,7 @@ module.exports = {
     sourceType: 'module'
   },
   rules: {
-    'jsdoc/check-examples': 0
+    'jsdoc/check-examples': 0,
+    'max-len': 0
```

Comment:
I am a -1 to disabling the max-length rule for the entire project. Instead, we should selectively disable it for specific lines when necessary and try to fix the issues wherever possible.

Reply:
linted all 98 of them (90 minutes of work to do that) here: https://github.com/digitalbazaar/vc-data-model-2-test-suite/pull/6/commits/5db71923170f8358aa5dcb1cf05c194146dd2bdc

Reply:
Thanks @aljones15! Sorry for the trouble 😁

Reply:
not any major trouble, but we do have a deadline so we might want to keep tasks a little more focused on the main goal of getting these test suites working.